include/beast/core/basic_streambuf.hpp

```diff
@@ -209,6 +209,36 @@ class basic_streambuf
         return this->member();
     }
 
+    /** Returns the default allocation size.
+
+        This is the smallest size that the stream buffer will allocate.
+        The size of the allocation can influence capacity, which will
+        affect algorithms that use capacity to efficiently read from
+        streams.
+    */
+    std::size_t
+    alloc_size() const
+    {
+        return alloc_size_;
+    }
+
+    /** Set the default allocation size.
+
+        This is the smallest size that the stream buffer will allocate.
+        The size of the allocation can influence capacity, which will
+        affect algorithms that use capacity to efficiently read from
+        streams.
+
+        @note This will not affect any already-existing allocations.
+
+        @param n The number of bytes.
+    */
+    void
+    alloc_size(std::size_t n)
```

Comment:
D'oh.  For others, it is [here](https://github.com/vinniefalco/Beast/pull/146/files/ebebe52612fc9a5f754ab4caf5ddb45b97b8f599#diff-9f4842e13a767b52e0c1c9e0437e6f1bR462)